net/gsocket/Makefile

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
```

Comment:
Omit or assign yourself.
According to [CONTRIBUTING](https://github.com/openwrt/packages/blob/8ece746b51f994ef74ca8ad20f5f05f370af2fbc/CONTRIBUTING.md#makefile-contents-should-contain).